src/port1.0/portfetch.tcl

```diff
@@ -607,11 +607,20 @@ proc portfetch::fetch_addfilestomap {filemapname} {
     }
 }
 
+# Utility function to error out if Xcode is needed
+proc portfetch::checkxcode {} {
```

Comment:
Why not do this in the existing procedure _check_xcode_version?

Reply:
Right, moved to `_check_xcode_version`